src/main/java/org/broadinstitute/hellbender/tools/exome/PerformSegmentation.java

```diff
@@ -48,6 +48,10 @@
  *   --output tumor.seg
  * </pre>
  *
+ * While the tangent normalized (tn) data are log2 transformed, as indicated by the --log2Input true option, 
```

Comment:
Perhaps: "While the tangent normalized (tn) input coverage may be log2 transformed, if indicated by the --log2Input true option..."

That is, PerformSegmentation will still work if you give non-log2 input (produced by some other tool besides NormalizeSomaticReadCounts) and specify --log2Input false, and the output will again be non-log2.